### PR TITLE
adding example using nvidia mps

### PR DESCRIPTION
**Pull request by weallcock**

### CODE REVIEW

File: docs/polaris/queueing-and-running-jobs/example-job-scripts.md [hunk 1]
```diff
@@ -107,3 +107,55 @@ The script is hard-coded for 4 GPUs on a Polaris node and simply pairs MPI ranks
 
 * IMPORTANT: If planning large-scale runs with many thousands of MPI ranks, then it is advised to comment out the `echo` command so as not to have thousands of lines of output written to stdout. 
 
+### Using MPS on the GPUs
+
+Documentation for the Nvidia Multi-Process Service (MPS) can be found [here](https://docs.nvidia.com/deploy/mps/index.html)
+
+In the script below, note that if you are going to run this as a multi-node job you will need to do this on every compute ndoe and you will need to ensure that the paths you specify for `CUDA_MPS_PIPE_DIRECTORY` and `CUDA_MPS_LOG_DIRECTORY`do not "collide" and end up with all the nodes writing to the same place.  The local SSDs or /dev/shm or incorporation of the node name into the path would all be possible ways of dealing with that issue.
```
Comment (cjknight) [inline docs/polaris/queueing-and-running-jobs/example-job-scripts.md:114]: small typo: "compute ndoe" -> "compute node"

**Review by cjknight — APPROVED** [review]
Code looks good. I was able to reproduce and also confirm with a LAMMPS runtime improvement when MPS enabled on single node.

**Comment by weallcock** [comment]
@bethcerny Adam wrote it and tested it and Chris also tested and confirmed it on LAMMPS, so if you can look at it for formatting and such and approve I think this should be good to merge.

File: docs/polaris/queueing-and-running-jobs/example-job-scripts.md (hunk 1 above)
Comment (atanikan) [inline docs/polaris/queueing-and-running-jobs/example-job-scripts.md:114]: typo ndoe to node

Comment (weallcock) [reply docs/polaris/queueing-and-running-jobs/example-job-scripts.md]: fixed with commit e6cc7cf

**Pull request merged**